### DISCUSSION TITLE
发布一款光谷社区第三方 Android App — 码志

### DISCUSSION THREAD

**Discussion by mzlogin** [mzlogin/blog-comments#223] [Announcements]
https://mazhuang.org/2017/10/14/guanggoo-android-app/ 

 在过去的一个来月，我利用业余时间做了一款光谷社区的第三方 Android 客户端。

**Comment by wakemind** [comment]
ddgd

**Comment by wakemind** [comment]
还得登录哟

**Comment by mzlogin** [comment]
> @wakemind
> 还得登录哟

嗯~社区网站对查看内容有一些限制规则，我这里实现时为了统一，直接将挺多资源设成不登录不让看了。

**Comment by HesitationPencil** [comment]
cv

**Comment by Appigle** [comment]
**mark 一波，原来光谷社区的app是你做的啊~ /大拇指**

**Comment by mzlogin** [comment]
> @Appigle
> **mark 一波，原来光谷社区的app是你做的啊~ /大拇指**

嗯嗯 惭愧惭愧 有好一阵没更新版本了~有什么意见和建议欢迎提哈
